R/pool.R

```diff
@@ -1,12 +1,47 @@
 
 
-#' TODO
+#' Pool Analysis Results
 #'
-#' @name pool
-#' @param results TODO
-#' @param conf.level TODO
-#' @param alternative TODO
-#' @param type TODO
+#' @param results an analysis object created by [analyse()]
+#'
+#' @param conf.level confidence level of the returned confidence interval.
+#' Must be a single number between 0 and 1. Default is 0.95
+#'
+#' @param alternative a character string specifying the alternative hypothesis,
+#' must be one of "two.sided" (default), "greater" or "less".
+#'
+#' @param type a character string of either `"percentile"` (default) or
+#' `"normal"`. Determines what method should be used to calculate the confidence
+#' intervals. See details.
+#' Only used if `method_condmean(type = "bootstrap")` was specified
+#' in the original call to [draws()]
+#'
+#' @param x A pool object generated by [pool()]
+#'
+#' @param ... not used
+#'
+#' @details
+#'
+#' The calculation used to generate the point estimate, standard errors and
+#' confidence interval depends upon the method specified in the original
+#' call to [draws()]; In particular:
+#'
+#' - `method_approxbayes()` & `method_bayes()` both use Rubin's rule to pool estimates
+#'  and variances across multiple imputed data, and the Barnard-Rubin rule to pool
+#'  degree's of freedom; see Little & Rubin (2002).
+#' - `method_condmean(type = "bootstrap")` Uses percentile or normal approximation;
+#' See Efron & Tibshirani (1994). Note that under this method no standard error is
+#' calculated and as such will be `NA` in the returned object / data.frame
+#' - `method_condmean(type = "jackknife")` Uses the standard jackknife formulas;
```

Comment:
formulas -> formula

Reply:
Maybe it is worth mentioning that when `method_condmean(type = "jackknife")` inference is based on normal approximation of the distribution of the treatment effect (as in `type = normal`)

Reply:
done